dali/test/python/test_pipeline.py

```diff
@@ -1154,27 +1171,22 @@ def define_graph(self):
     pipe.release_outputs()
     pipe.schedule_run()
     pipe.outputs()
-    try:
+    with assert_raises(RuntimeError):
         pipe.run()
-        assert(False)
-    except RuntimeError:
-        assert(True)
     # disable check
     pipe.enable_api_check(False)
-    try:
-        pipe.run()
-        assert(True)
-    except RuntimeError:
-        assert(False)
+    pipe.run()
```

Comment:
missing `assert_raises`

Reply:
As with the two above cases, the code seems to be a very verbose way of saying that it should not raise.